target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf289f.dts

```diff
@@ -0,0 +1,436 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2022, Pawel Dembicki <paweldembicki@gmail.com>.
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+        model = "ZTE MF289F";
+        compatible = "zte,mf289f";
+
+        aliases {
+                led-boot = &led_internal;
+                led-failsafe = &led_internal;
+                led-running = &led_internal;
+                led-upgrade = &led_internal;
+        };
+
+        chosen {
+                /*
+                 * bootargs forced by u-boot bootipq command:
+                 * 'ubi.mtd=rootfs root=mtd:ubi_rootfs rootfstype=squashfs rootwait'
+                 */
+                bootargs-append = " root=/dev/ubiblock0_1";
+        };
+
+        gpio-restart {
+                compatible = "gpio-restart";
+                gpios = <&tlmm 8 GPIO_ACTIVE_HIGH>;
+        };
+
+        leds {
+                compatible = "gpio-leds";
+
+                led_internal: led-0 {
+                        function = LED_FUNCTION_STATUS;
+                        color = <LED_COLOR_ID_BLUE>;
+                        gpios = <&tlmm 10 GPIO_ACTIVE_LOW>;
+                        label = "blue:internal_led";

```

Comment:
You cannot use labels and color/function properties at the same time, label will override those.
"blue:internal_led" is not a valid name anyway if its used for status

Reply:
As base dts I've used one of MF286D (most similar to this one).

So I've to remove color? Because internal led is only blue (and cannot change colors)

Reply:
@robimarko this is the only LED available for status indeed. Front panel LEDs are controlled by modem.
In MF286{,A,R} I called it blue:debug out of lack of better idea, @CHKDSK88 chose that name in MF286D, I would stay with that.

Reply:
No, dont remove the color.
If it's used for status only, just remove the label, and it will be blue:status by default.

@Leo-PL Debug makes no sense if its actually used as the OpenWrt status LED.

Reply:
ok, removed